backend/battles/battle.py

```diff
@@ -0,0 +1,45 @@
+from .email import send_battle_result  # noqa
+
+
+def win_attack(pokemon_1, pokemon_2):  # pokemon_1 always attacks and pokemon_2 always defends
+    if pokemon_1.attack > pokemon_2.defense:
+        return True
+    return False
```

Comment:
You can simplify this with:
```suggestion
def win_attack(pokemon_1, pokemon_2):  # pokemon_1 always attacks and pokemon_2 always defends
    return pokemon_1.attack > pokemon_2.defense
```

Reply:
Also, I don't feel this function name is clear. Imagine if you couldn't see the function body, would reading just `win_attack` be clear to you what it is about? Usually we name functions starting with a verb and then what it's about, because it can be `get/fetch`, `delete`, `update`, etc.

Maybe naming it `compare_attack_to_defense` would be clearer, but it's just a suggestion, you can name it whatever you want.